config/initializers/permissions.rb

```diff
@@ -217,6 +217,8 @@
                        work_packages: %i[show index],
                        work_packages_api: [:get],
                        "work_packages/reports": %i[report report_details],
+                       "work_packages/activities_tab": %i[index create update_streams edit cancel_edit update
```

Comment:
`create edit update`  etc shouldn't be part of the view_permission

Reply:
Done